wordle-web/pages/game.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <v-row justify="center" align="center">
+    <v-card color="slate" ripple>
+      <v-card-title class="display-1">
+        Welcome to the Enigma Vocabulary!
+      </v-card-title>
+      <v-card-text>
+        <p class="headline">
+          I want to play a game!
+        </p>
+        <p class="body-1">
+          Here are the rules:<br>
+          1)Your guess has to be a real word. No random letters.<br>
+          2)Gray letters are not found in the word at all.<br>
+          3)Yellow letter are correct guesses but in the wrong spot.<br>
+          4)Green letters are correct guesses in the correct spot. Don't move them.<br>
+          5)Words can have multiples of the same letter, ex: Apple<br>
+          6)You have {{maxTries}} tries to guess the correct word.<br>
+        </p>
+        <p class="display-1">
+          Now let's play a game!
+        </p>
```

Comment:
No need to wrap these in `<p>` tags. Instead, you should wrap them in their on individual `<v-card-text>` tags like so:
```suggestion
        <v-card-text class="headline">
          I want to play a game!
        </v-card-text>
        <v-card-text class="body-1">
          Here are the rules:<br>
          1)Your guess has to be a real word. No random letters.<br>
          2)Gray letters are not found in the word at all.<br>
          3)Yellow letter are correct guesses but in the wrong spot.<br>
          4)Green letters are correct guesses in the correct spot. Don't move them.<br>
          5)Words can have multiples of the same letter, ex: Apple<br>
          6)You have {{maxTries}} tries to guess the correct word.<br>
        </v-card-text>
        <v-card-text class="display-1">
          Now let's play a game!
        </v-card-text>
```